### ISSUE TITLE
Adblock not working

### ISSUE THREAD

**Issue by tf330129** [reek/anti-adblock-killer#4610]
Hello. Two days ago this script stopped working and YouTube is once again detecting uBlock Origin.

**Comment by TecnikOfficial** [comment]
try its [fork](https://github.com/bogachenko/fuckfuckadblock) or use [enchancer for youtube](https://www.mrfdev.com/enhancer-for-youtube) extension it has adblock designed only for youtube and its robust

**Comment by ghost** [comment]
https://www.reddit.com/r/uBlockOrigin/comments/169iri2/youtube_antiadblock_and_ads_september_04_2023/

**Comment by ghost** [comment]
@techgamers2 the first project is blocked by uBlock Origin and Enhancer for YouTube allegedly gets detected by YouTube. uBlock Origin volunteers address YouTube anti-adblock periodically, so just update its default filter lists and don't use any other ad blocker.

**Comment by TecnikOfficial** [comment]
Yes now enhancer for yt is not working but when i posted my initial comment
2 months ago it ws working .. Also for my default ublock filters sometimes
blocks tht anti-adblock yt banner but sometimes doesnt so i prefer to use
tht anti-adblock fork filterlist i previously mentioned (btw its filterlist
not another adblocker) so all the time i never get tht anti-adblock yt
banner plus it provides all over protection for a lot of sites to prevent
adblock detection and as for enhancer for yt i use it for its other
features but for the time being i disabled its block ads option as
mentioned by its dev
